runtime-k3d/src/runtime/config/helm/values.yaml

```diff
@@ -1,62 +1,47 @@
-# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
-#
-# This program and the accompanying materials are made available under the
-# terms of the Apache License, Version 2.0 which is available at
-# https://www.apache.org/licenses/LICENSE-2.0.
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations
-# under the License.
-#
-# SPDX-License-Identifier: Apache-2.0
-
-imageSeatService:
-  name: seatservice
-  repository: k3d-registry.localhost:12345/seatservice
-  pullPolicy: Always
-  daprAppid: seatservice
-  daprAppPort: 50051
-  daprLogLevel: debug
+imageMosquitto:
+  args:
+  - mosquitto
+  - -c
+  - /mosquitto-no-auth.conf
+  name: mqtt-broker
+  nodePort: '{{.Values.imageMosquitto.nodePort}}'
+  nodePort1: '{{.Values.imageMosquitto.nodePort1}}'
+  port: 1883
+  port1: '{{.Values.imageMosquitto.port1}}'
+  pullPolicy: IfNotPresent
+  repository: eclipse-mosquitto
   tag: latest
-  nodePort: 30051
-
+  targetPort: '{{.Values.imageMosquitto.targetPort}}'
+  targetPort1: '{{.Values.imageMosquitto.targetPort1}}'
+---
 imageVehicleDataBroker:
+  daprLogLevel: debug
+  kuksadatabrokermetadatafile: /vss.json
   name: vehicledatabroker
-  repository: k3d-registry.localhost:12345/vehicledatabroker
+  nodePort: '{{.Values.imageVehicleDataBroker.nodePort}}'
+  port: 55555
   pullPolicy: Always
-  daprAppid: vehicledatabroker
-  daprAppPort: 55555
+  repository: ghcr.io/eclipse/kuksa.val/databroker
+  tag: 0.3.0
+  targetPort: '{{.Values.imageVehicleDataBroker.targetPort}}'
+---
+imageSeatService:
   daprLogLevel: debug
-  tag: latest
-  nodePort: 30555
-
-imageMosquitto:
-  name: mqtt-broker
-  repository: eclipse-mosquitto
-  pullPolicy: IfNotPresent
-  args:
-    - mosquitto
-    - -c
-    - /mosquitto-no-auth.conf
-  tag: 2.0.14
-  port: 1883
-  nodePort: 31883
-  websocketPort: 9001
-
+  name: seatservice
+  nodePort: '{{.Values.imageSeatService.nodePort}}'
+  port: 50051
+  pullPolicy: Always
+  repository: ghcr.io/eclipse/kuksa.val.services/seat_service
+  tag: v0.2.0
+  targetPort: '{{.Values.imageSeatService.targetPort}}'
+---
 imageFeederCan:
+  candumpfile: /data/candumpDefault.log
+  daprLogLevel: debug
+  dbcfile: /data/dbcfileDefault.dbc
+  mapping: /data/mappingDefault.yml
+  mappingfile: /data/mappingDefault.yml
   name: feedercan
-  repository: k3d-registry.localhost:12345/feedercan
   pullPolicy: Always
-  daprAppid: feedercan
-  daprLogLevel: debug
-  candumpfile: "/data/candumpDefault.log"
-  dbcfile: "/data/dbcfileDefault.dbc"
-  mapping: "/data/mappingDefault.yml"
-  tag: latest
-
-nameOverride: ""
-fullnameOverride: ""
-
-vspecFilePath: ""
+  repository: ghcr.io/eclipse/kuksa.val.feeders/dbc2val
+  tag: v0.1.1
```

Comment:
shouldn't all the values be taken from the runtime.json?

Reply:
What do you mean by that, all available values are applied from runtime.json, other values are used from the template itself.

Reply:
I mean we don't need this template at all if we have the values already in the runtime.json

Reply:
Currently, there are only two fields we don't have in runtime.json, they are pullPolicy and daprLogLevel, if we decide to put then in runtime.json or use some default values for them, we then do not need template anymore.

Reply:
I would recommend doing this since we have a single point of truth then, @doosuu what do you think?

Reply:
or we simply hardcode them for now and if it is needed we can add it later to the runtime